bitbots_odometry/launch/odometry.launch

```diff
@@ -1,3 +1,4 @@
 <launch>
+    <node name="motion_odometry" pkg="bitbots_odometry" type="motion_odometry" />
```

Comment:
I'm a bit concerned about this... do we have to remove /rename the walk odometry in other packages?

Reply:
yes we have to always launch the new odometry.launch file now.
I think it make semantically more sense to have one odometry launch file which launches all nodes necessary for the odometry